masonry/src/lib.rs

```diff
@@ -178,7 +178,7 @@ pub use cursor_icon::{CursorIcon, ParseError as CursorIconParseError};
 pub use kurbo::{Affine, Insets, Point, Rect, Size, Vec2};
 pub use parley::layout::Alignment as TextAlignment;
 pub use parley::style::FontWeight;
-pub use vello::peniko::{Color, Gradient};
+pub use vello::peniko::{color::palette, Color, Gradient};
```

Comment:
For reviewers, this is the new change to public API (it wasn't obvious to me on a first pass).

Reply:
And a similar in `xilem/src/lib.rs` re-exporting from here (as it already did for `Color`).